app/src/store/configureStore.js

```diff
@@ -0,0 +1,36 @@
+import { createStore, applyMiddleware, compose, combineReducers } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension/logOnlyInProduction';
+import { connectRoutes } from 'redux-first-router';
+import queryString from 'qs';
+import reduxThunk from 'redux-thunk';
+
+import routesMap, { onBeforeRouteChange } from 'routes/routesMap';
+import reducers from './reducers';
+
+const composeEnhancers = (...args) =>
+  typeof window !== 'undefined' ? composeWithDevTools({})(...args) : compose(...args);
+
+export const configureStore = (history, preloadedState) => {
+  const { reducer, middleware, enhancer } = connectRoutes(history, routesMap, {
+    querySerializer: queryString,
+    onBeforeChange: onBeforeRouteChange,
+  });
+
+  const reduxDevToolsExtension = window.__REDUX_DEVTOOLS_EXTENSION__; // eslint-disable-line no-underscore-dangle
+  const reduxLogger = reduxDevToolsExtension ? reduxDevToolsExtension({ maxAge: 30 }) : (f) => f;
```

Comment:
I get an error about multiple initialization of the redux devtools. I guess these 2 lines can be removed now.